proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -295,3 +298,14 @@ message ConsumerAddrsToPrune {
   uint64 vsc_id = 2;
   AddressList consumer_addrs = 3;
 }
+
+// EpochValidator is used to facilitate epoch-based transitions. It contains relevant info for
+// a validator that is opted in, in an epoch, on a consumer chain.
+message EpochValidator {
+  // validator's consensus address on the provider chain
+  bytes provider_cons_addr = 1;
+  // voting power the validator has during this epoch
+  int64 power = 2;
+  // public key the validator uses on the consumer chain during this epoch
+  bytes consumer_public_key = 3;
```

Comment:
Changed to:
```
 tendermint.crypto.PublicKey consumer_public_key = 3;
```
similarly to how a public key is declared in `message ValidatorConsumerPubKey`. This actually simplified things a lot because we do not have to marshal and unmarshal all the time.

Reply:
Let's just make sure it works with multisig keys -- see https://github.com/cosmos/interchain-security/releases/tag/v1.1.0-multiden for more details.

Reply:
The provided [link](https://github.com/cosmos/interchain-security/releases/tag/v1.1.0-multiden) points to an issue about the type of the public key in the `MsgAssignConsumerKey` transaction. In our case `ConsumerValidator`s are only set internally so I don't see this being an issue. Also, note that the [`MsgOptIn`](https://github.com/cosmos/interchain-security/pull/1620/files) message gets the consumer public key as a `string` similarly to `MsgAssignConsumerKey`.